src/org/zanata/jenkins/Reporting.groovy

```diff
@@ -0,0 +1,30 @@
+package org.zanata.jenkins
+import org.zanata.jenkins.ScmGit
+import hudson.AbortException
+import com.cloudbees.groovy.cps.NonCPS
+
+
+class Reporting implements Serializable {
+
+  // Send test coverage data to codecov.io
+  @NonCPS
+  static void codecov(def env, def steps, String repoUrl){
+    ScmGit scmGit = new ScmGit(env, steps, repoUrl)
+    String sha = scmGit.getCommitId(env.BRANCH_NAME)
+    try {
+      withCredentials(
+        [[$class: 'StringBinding',
+        credentialsId: 'codecov_zanata-platform',
+        variable: 'CODECOV_TOKEN']]) {
+        // NB the codecov script uses CODECOV_TOKEN
+        steps.sh "curl -s https://codecov.io/bash | bash -s - -K -B ${env.BRANCH_NAME} -C ${sha} -P ${env.CHANGE_ID} -b ${env.BUILD_NUMBER}"
```

Comment:
Could you please add a comment here, something like this?

    // TODO use checkout.GIT_COMMIT (https://zanata.atlassian.net/browse/ZNTA-2237)